src/Header/Header.jsx

```diff
@@ -95,7 +95,7 @@ export class Header extends Component {
 						}}
 					</HeaderContext.Consumer>
 
-					<div className={styles.header} data-tracking="Global nav">
+					<div className={styles.header} data-tracking="Global nav" id="top">
```

Comment:
Should probably have a test that asserts that we have a back to top scroll target id